tsconfig.json

```diff
@@ -21,5 +21,5 @@
     "declaration": true
   },
   "exclude": ["node_modules"],
-  "include": ["./**/ts"]
+  "include": ["./**/ts", "cli/tests/unit/cli-genMaciKeypair.test.ts", "cli/tests/unit/genPubkey.test.ts"]
```

Comment:
Why do these tests included in bundle?

Reply:
I have no idea 🤔 I dragged them over to cli/tests before changing name, maybe vs code added them for whatever reason? so odd haha will remove tho